pkg/loop/server.go

```diff
@@ -90,12 +90,15 @@ func (s *Server) start() error {
 		if tracingConfig.Enabled {
 			attributes = tracingConfig.Attributes()
 		}
+
 		beholderCfg := beholder.Config{
 			InsecureConnection:       envCfg.TelemetryInsecureConnection,
 			CACertFile:               envCfg.TelemetryCACertFile,
 			OtelExporterGRPCEndpoint: envCfg.TelemetryEndpoint,
 			ResourceAttributes:       append(attributes, envCfg.TelemetryAttributes.AsStringAttributes()...),
 			TraceSampleRatio:         envCfg.TelemetryTraceSampleRatio,
+			AuthHeaders:              envCfg.TelemetryAuthHeaders,
```

Comment:
Sorry, is it clear how AuthHeaders are created? I.e., is one supposed to call the BuildAuthHeaders function with via some command line tool prior starting the node?

Reply:
Ok, it is done here https://github.com/smartcontractkit/chainlink/pull/15040/.